app/models/spree_adyen/gateway/payment_sessions.rb

```
@@ -102,8 +108,24 @@ def complete_payment_session(payment_session:, params: {})
 
       private
 
+      # Resolves a sessionResult from a redirectResult by calling Adyen's /payments/details endpoint.
+      # In the sessions flow, Adyen processes the redirect server-side. We call /payments/details
+      # with the redirectResult to finalize the payment and get the sessionResult.
+      def resolve_session_result_from_redirect(payment_session, redirect_result)
```

RuboCop (1.86.1)

[warning] Unused method argument - `payment_session`. If it's necessary, use `_` or `_payment_session` as an argument name to indicate that it won't be used. If it's unnecessary, remove it.

(Lint/UnusedMethodArgument)